qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -20,8 +20,8 @@ class Env {
             "ROX_USERNAME": "admin",
     ]
 
-    static final IN_CI = (System.getenv("CI") != null)
-    static final CI_JOBNAME = System.getenv("CI_JOB_NAME") ?: System.getenv("CIRCLE_JOB") ?: ""
```

Comment:
Searching the repo for `CIRCLE_JOB` reveals few more interesting places worth to clean up.
Keywords: `CIRCLE_JOB`, `is_CIRCLECI`, `CIRCLE_BUILD_NUM`, `CIRCLE_PULL_REQUEST`.

Reply:
Yeah. Probably a more complete cleanup job to scope.